apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -50,6 +50,8 @@ const EVENTS = {
   CAP_AGE_GATED_MODAL_SHOWN: 'CAP Teacher Students Warning Modal Shown',
   CAP_STUDENT_WARNING_LINK_CLICKED:
     'CAP Teacher Students Warning Modal Documentation Clicked',
+  CAP_AGE_GATED_BANNER_SHOWN: 'CAP Teacher Students Banner Shown',
+  CAP_AGE_GATED_BANNER_CLICKED: 'CAP Teacher Students Banner Clicked',
```

Comment:
As I was testing, I realized that the `Banner Clicked` is measuring the same as the `Student Modal Shown`:

```
[AMPLITUDE ANALYTICS EVENT]: CAP Teacher Students Banner Clicked. Payload: {"payload":{"user_id":1,"number_of_gateable_students":1}}
[AMPLITUDE ANALYTICS EVENT]: CAP Teacher Students Warning Modal Shown. Payload: {"payload":{"user_id":1,"number_of_gateable_students":1}}
```

Reply:
Correct, I realized this may be redundant during implementation but kept it in as it is a Metric that is asked for in the Jira [task](https://codedotorg.atlassian.net/jira/software/c/projects/P20/boards/69?assignee=712020%3Af0e2716f-09f9-4686-8490-9f1f9a08bb06&selectedIssue=P20-904)